nemo-physical/src/datatypes/data_value.rs

```diff
@@ -63,3 +63,20 @@ impl std::fmt::Display for DataValueT {
         }
     }
 }
+
+/// Iterator over one kind of possible data values
+#[allow(missing_debug_implementations)]
```

Comment:
I guess a small handwritten `Debug` implementation couldn't hurt (it would just state which variant it's an iterator over), but for now this is also okay.